code/modules/surgery/eye_surgery.dm

```diff
@@ -74,3 +73,15 @@
 		)
 		display_pain(target, "You feel a visceral stabbing pain right through your head!") // dunno who can feel pain w/o a brain but may as well be consistent.
 	return FALSE
+
+/datum/surgery/eye_surgery/mechanic
```

Comment:
modularized `monkestation/code/modules/surgery/eye_surgery.dm`- yeah i think you get the point by now